### PR TITLE
Add CAP Docs

### PR DESCRIPTION
**Pull request by nwmac**
Document install process for CAP - including Metrics and configuration

### CODE REVIEW

File: docs/suse/deploy.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+- Deploy Stratos Metrics
+- Connect Stratos Metrics to Stratos to enable Metrics views in Stratos
+
+These instructions assume that you are deploying the SUSE Cloud Application Platform and that you have deployed UAA/SCF using a `scf-config-values` configuration file.
+
+## Install Stratos with Helm
+
+To deploy Stratos, use the same `scf-config-values.yaml` file that you used to deploy SCF/UAA.
+
+Ensure you have a Storage Class configured and that is set to be the default.
```
Comment (richard-cox) [inline docs/suse/deploy.md]: Do we have any docs anywhere that might give hints on this?

File: docs/suse/deploy.md [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+1. Check the `Skip SSL validation for the endpoint` if using self-signed certificates
+1. Click `Finish`
+
+The view will refresh to show the new endpoint in the disconnected state.
+
+Next you will need to connect to this endpoint.
+
+1. In the table of endpoints, click the three-dot menu icon alongside the endpoint that you added above
+1. Click on 'Connect' in the dropdown menu
+1. Select the appropriate authentication type and provide the necessary values
```
Comment (richard-cox) [inline docs/suse/deploy.md]: Did @irfanhabib write some docs on these?

File: docs/suse/deploy.md [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+The view will refresh to show the new endpoint in the disconnected state.
+
+Next you will need to connect to this endpoint.
+
+1. In the table of endpoints, click the three-dot menu icon alongside the endpoint that you added above
+1. Click on 'Connect' in the dropdown menu
+1. Enter the username and password for your Stratos Metrics instance
+1. Click `Connect`
+
+Once connected, you should see that the name of your Metrices endpoint is a hyperlink and clicking on it should show basic metadata about the Stratos Metrics endpoint.
```
Comment (richard-cox) [inline docs/suse/deploy.md:160]: It it worth giving some examples of metrics for user to prove out it's getting data (cf app + pod metrics)?

File: docs/suse/deploy.md (hunk 2 above)
Comment (nwmac) [reply docs/suse/deploy.md]: Not sure - I was only covering CaaSP for now - I could add notes for others

File: docs/suse/deploy.md (hunk 3 above)
Comment (nwmac) [reply docs/suse/deploy.md:160]: Yep

**Review by richard-cox — APPROVED** [review]

**Pull request merged**